styles.css

```diff
@@ -379,3 +387,182 @@ form input[type=submit] {
   text-align: center;
   color: #42526e;
 }
+
+/* Media Queries */
+
+@media only screen and (min-width: 768px) {
+  body {
+    background:
+      url(images/desktopBg.png) no-repeat,
+      url(images/deskFormBg1.png) no-repeat left 0% top 3422px,
+      url(images/deskFormBg2.png) no-repeat left 19% top 3584px,
+      url(images/deskFormBg3.png) no-repeat right 0% top 3278.3px;
+    background-size: cover, 200px, 102px, 214px;
+  }
+
+  header nav,
+  header a {
+    display: none;
+  }
+
+  header ul {
+    display: flex;
+    position: absolute;
+    top: 16px;
+    left: 50%;
+    transform: translate(-50%);
+    text-align: center;
+    align-items: center;
+    justify-content: space-between;
+    width: 255px;
+  }
+
+  .intro {
+    all: unset;
+    position: absolute;
+    top: 232px;
+    left: 50%;
+    transform: translate(-50%);
+    width: 63.89%;
+    background-color: transparent;
+  }
+
+  .intro h1 {
+    all: unset;
+    height: 114px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    text-align: center;
+    letter-spacing: 0.37px;
+    background-color: transparent;
+  }
+
+  .intro p {
+    all: unset;
+    height: 96px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    background-color: transparent;
+  }
+
+  #line-break::after {
+    content: "\a";
+    white-space: pre;
+  }
+
+  #line-break::before {
+    content: "\a";
+    white-space: pre;
+  }
+
+  .icons {
+    position: absolute;
+    top: 490px;
+    background-color: transparent;
+  }
+
+  .action {
+    position: absolute;
+    top: 604px;
+  }
+
+  #portfolio {
+    position: absolute;
+    top: 1139px;
+    left: 50%;
+    transform: translate(-50%);
+    grid-template-columns: 33% 33% 33%;
+    width: 1171px;
+    column-gap: 0;
+    background-color: transparent;
+  }
+
+  #portfolio h3 {
+    grid-column-start: 1;
+    grid-column-end: 4;
+  }
+
+  #about {
+    position: absolute;
+    top: 2411px;
+    left: 50%;
+    transform: translate(-50%);
+    width: 100%;
+  }
```

Comment:
You did an awesome job working on your desktop version! 💯 However, there's still some room to improve in this section:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/208134757-8d67ff8b-3e03-4e1c-91bb-e4b9bbf76d14.png) | ![image](https://user-images.githubusercontent.com/77069011/208134607-7ae449bc-73fc-4060-b376-0103cf39aa66.png) |

- [ ] Please consider adding more horizontal spacing to the edges of this section, at the moment, the text overlaps with the background image! 🚧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/208135198-a14e02df-741a-47e0-a38a-d09bc7d9aac9.png) | ![image](https://user-images.githubusercontent.com/77069011/208135061-17da9fd6-5d3c-47ce-b28a-79930faf29e4.png) |

- [ ] The "Get my resume" button overlaps your paragraph text in the mobile version. Please dig into this and fix it! 🚧

Please consider applying these changes! 🤓 🛠️